app/models.py

```diff
@@ -591,57 +591,62 @@ def create(cls, email, name="", password=None, from_partner=False, **kwargs):
 
         return user
 
-    # region Billing
-    def lifetime_or_active_subscription(self) -> bool:
-        """True if user has lifetime licence or active subscription"""
-        if self.lifetime:
-            return True
-
+    def get_active_subscription(
```

Comment:
not related to the double subscription email but I think we can rename User.get_subscription to something like User.get_paddle_subscription to avoid confusion with this new method.

Reply:
:+1: